ste/xfer-anyToRemote.go

```diff
@@ -314,16 +314,15 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 	s.Epilogue() // Perform service-specific cleanup before jptm cleanup. Some services may actually require setup to make the file actually appear.
 
 	if jptm.IsLive() && info.DestLengthValidation {
-		if s2sc, isS2SCopier := s.(s2sCopier); isS2SCopier { // TODO: Implement this for upload and download?
-			destLength, err := s2sc.GetDestinationLength()
+		_, isS2SCopier := s.(s2sCopier)
+		destLength, err := s.GetDestinationLength()
 
-			if err != nil {
-				jptm.FailActiveSend("S2S Length check: Get destination length", err)
-			}
+		if err != nil {
+			jptm.FailActiveSend(common.IffString(isS2SCopier, "S2S ", "Download ")+"Length check: Get destination length", err)
```

Comment:
>Download [](start = 62, length = 8)

shouldn't it be Upload?

Reply:
approved with comment

Reply:
Yes, it should be.  Or just omit it, I think, and let FailActiveSend provide the contxt (upload/sys).